toolkits/landcover/impl/landsat8.js

```diff
@@ -31,6 +31,24 @@ var COMMON_BAND_NAMES = {
   'B11': 'thermal2'
 };
 
+/**
+ * The TasseledCap transformation specific to Landsat 8.
+ *
+ * Muhammad Hasan Ali Baig, Lifu Zhang, Tong Shuai & Qingxi Tong (2014)
+ * Derivation of a tasselled cap transformation based on Landsat 8 at-satellite
+ * reflectance, Remote Sensing Letters, DOI: 10.1080/2150704X.2014.915434
+ *
+ * Coefficients are in the order: [blue, green, red, nir, swir1, swir2]
```

Comment:
Add `@override`

Reply:
Done, but it doesn't actually override anything (there isn't one in the parent class).